packages/data-point/lib/core/__snapshots__/core.test.js.snap

```diff
@@ -0,0 +1,17 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
I added this test to check against the API we are exposing on dataPoint instances